nsqlookupd/options.go

```diff
@@ -16,7 +18,7 @@ type Options struct {
 	InactiveProducerTimeout time.Duration `flag:"inactive-producer-timeout"`
 	TombstoneLifetime       time.Duration `flag:"tombstone-lifetime"`
 
-	Logger logger
+	Logger app.Logger
```

Comment:
Any other idea for providing the common types w/o duplicating the same within each of the public packages?